src/assets/base/base.scss

```diff
@@ -1,18 +1,257 @@
 @import "/src/assets/styles/abstracts/mixins.scss";
 @import "/src/assets/themes/default.scss";
 
-.card-header {
-    background-color: $primary;
-    color: $card-header-color ;
-}
-
-.park-select-header {
-    background-color: $primary;
-    color: $header-font-color;
+hr {
+    border: none;
 }
 
 .registration-details-container {
     background-color: $feature-block-bg;
     border-radius: 5px;
     color: white;
 }
+
+.main-wrapper {
+    display: table;
+    height: calc( 100% - #{$header-hight-mobile} - #{$footer-hight-mobile} );
+    width: 100%;
+    background: $bg-default;
+}
+
+.main-content {
+    background-color: white;
+    height: 100%;
+    margin-bottom: 120px;
+}
+
+// buttons
+
+.BC-Gov-PrimaryButton {
```

Comment:
button styling classes should overrule bootstrap `btn-*` classes - `btn-primary`, `btn-secondary`, etc as per mark's comment (typ)